### DISCUSSION TITLE
What about a cheap webcam?

### DISCUSSION THREAD

**Discussion by bcjmk** [jomjol/AI-on-the-edge-device#2508] [General]
Solar powered webcams are dropping in price - total kit is ~$30 (27 EU). My meter gets plenty of sunlight, so that's a benefit. Can we pass the detection zones into the AI processor? Don't know what SW these cameras use, but I've generally had good luck getting the rtsp feeds into HA.

https://www.amazon.com/Aiwit-Wireless-Detection-Waterproof-Real-Time/dp/B0C49GR318/ref=sr_1_5?keywords=solar+cameras&qid=1688903106&sr=8-5

**Comment by caco3** [comment]
This is not within the scope of this project!
